go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/pingcap/TiProxy
 
-go 1.19
+go 1.18
```

Comment:
Why?

Reply:
My another laptop is `1.18`. I modified it to pass my local test. Never mind, it will be modified back.

Reply:
Done.